src/it/scala/com/exasol/cloudetl/kinesis/KinesisAbstractIntegrationTest.scala

```diff
@@ -0,0 +1,142 @@
+package com.exasol.cloudetl.kinesis
+
+import java.io.File
+import java.nio.file.Path
+import java.sql.ResultSet
+
+import com.exasol.cloudetl.kinesis.KinesisConstants._
+import com.amazonaws.SDKGlobalConfiguration.AWS_CBOR_DISABLE_SYSTEM_PROPERTY
+import com.amazonaws.services.kinesis.{AmazonKinesis, AmazonKinesisClientBuilder}
+import com.exasol.cloudetl.data.Row
+import com.exasol.containers.{ExasolContainer, ExasolContainerConstants}
```

Comment:
```suggestion
import com.exasol.cloudetl.data.Row
import com.exasol.cloudetl.kinesis.KinesisConstants._
import com.exasol.containers.{ExasolContainer, ExasolContainerConstants}

import com.amazonaws.SDKGlobalConfiguration.AWS_CBOR_DISABLE_SYSTEM_PROPERTY
import com.amazonaws.services.kinesis.{AmazonKinesis, AmazonKinesisClientBuilder}
```

Reply:
They will be reshuffled again after I reformat code in IDE. Maybe I can set the rules for an order, but I don't really know what are the rules. I don't add impotst by hands, Intellij does it and also orders imports

Reply:
Ahh okay, I see. Maybe, Intellij can use the settings from [style-config](https://github.com/exasol/cloud-storage-etl-udfs/blob/master/project/scalastyle-config.xml#L336-L345) file.
Rules are simple: 
- There are some groups, in order: `java`, `scala`, `com.exasol`, `others`.
- Withing each group imports should be ordered lexicographical.

Reply:
I didn't find a way to use them :(